components/description/description.module.scss

```diff
@@ -0,0 +1,55 @@
+$colors: #456c3d, #669f54, #7ec16a, #b9e7ac, #ecedf0;
```

Comment:
Please add this array to the colors file and import it from there: https://github.com/urish/pull-request-community/blob/master/styles/colors.ts